CHANGELOG/CHANGELOG-1.32.md

```diff
@@ -1070,13 +1070,6 @@ name | architectures
 
 ### (No, really, you MUST read this before you upgrade)
```

Comment:
Maybe drop this heading, then?

Reply:
`## Urgent Upgrade Notes` should be removed as well

Reply:
SGTM, I wasn't sure about that with the standardized layout. You could argue that it's more or less re-assuring to have the section empty vs missing and I didn't think a lot about that yet.

Dropping.